akka-actor/src/main/scala/akka/actor/AbstractFSM.scala

```diff
@@ -388,6 +421,17 @@ abstract class AbstractFSM[S, D] extends FSM[S, D] {
   final def setTimer(name: String, msg: Any, timeout: FiniteDuration): Unit =
     setTimer(name, msg, timeout, false)
 
+  /**
+   * Schedule named timer to deliver message after given delay, possibly repeating.
+   * Any existing timer with the same name will automatically be canceled before
+   * adding the new timer.
+   * @param name identifier to be used with cancelTimer()
+   * @param msg message to be delivered
+   * @param timeout delay of first message delivery and between subsequent messages
+   */
+  final def setTimer(name: String, msg: Any, timeout: java.time.Duration): Unit =
+    setTimer(name, msg, JavaDurationConverters.JavaDurationOps(timeout).asScala, false)
```

Comment:
Missing the four parameter version of the metod.

Reply:
would also be better to delegate to the `def setTimer(name: String, msg: Any, timeout: FiniteDuration)` to avoid having the default value `false` at two places